website/pages/_meta.ts

```diff
@@ -20,11 +20,16 @@ const meta = {
   'oneof-input-objects': 'OneOf input objects',
   'defer-stream': '',
   '-- 3': {
+    type: 'separator',
+    title: 'API',
+  },
+  'api-v16': 'v16',
+  '-- 4': {
     type: 'separator',
     title: 'FAQ',
   },
   'going-to-production': '',
-  'api-v16': {
+  'api-menu': {
```

Comment:
The goal was to not so this, need to reach out to Dima on how to reset this context.

Reply:
Is the final goal to have the subheadings within the API show up somewhere?

Reply:
Only when you surf to the api docs else that would not scale to i.e. also having v17